Source/CharismaPlugNPlay/Public/Playthrough/CharismaPlaythroughData.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 Charisma Entertainment Ltd
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/NoExportTypes.h"
+#include "CoreMinimal.h"
+#include "MetaDataFunctionBase.h"
+#include "PlaythroughEntities.h"
+#include "Playthrough/CharismaConnectionParams.h"
+#include "Playthrough/CharismaPlayerComponent.h"
+#include "CharismaPlaythroughData.generated.h"
+
+/**
+ * 
+ */
+UCLASS(BlueprintType, Blueprintable, Category="Charisma")
+class CHARISMAPLUGNPLAY_API UCharismaPlaythroughData : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	UFUNCTION(BlueprintCallable, Category="Charisma|GameMode")
+	void CollateAllCharismaEntities(TArray<TSubclassOf<UMetaDataFunctionBase>> MetaClasses);
+	
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category="Charisma")
+	FCharismaConnectionParams ConnectionParams;
```

Comment:
![image](https://github.com/charisma-ai/unreal-plug-n-play/assets/132349556/fe7d7c14-3eeb-426f-bb5e-6cd6dbf2e554)

I don't see any way of setting class params w/o manually doing visual scripting. Would you be able to update the connection params so they're assigned in the GameMode BP and those values exposed in Class Defaults?